core/sup_tree_core/periodic_log/message_builder.ex

```diff
@@ -0,0 +1,32 @@
+# Copyright(c) 2015-2019 ACCESS CO., LTD. All rights reserved.
+
+defmodule AntikytheraCore.PeriodicLog.MessageBuilder do
+  @max_proc_to_log     5
+  @max_msg_to_log      10
+  @queue_len_threshold 100
+
+  def init() do
+    nil
+  end
+
+  def build_log(state) do
+    procs =
+      :recon.proc_count(:message_queue_len, @max_proc_to_log)
+      |> Enum.filter(fn({_pid, qlen, _info}) -> qlen >= @queue_len_threshold end)
+    if procs != [] do
+      log_time = Antikythera.Time.to_iso_timestamp(Antikythera.Time.now())
+      log =
+        procs
+        |> Enum.reduce(log_time, fn({pid, qlen, info}, acc) ->
+          acc2 = acc <> "\n" <> Integer.to_string(qlen) <> " " <> inspect(info, structs: false)
+          Process.info(pid)
```

Comment:
I thought that `Process.info(pid)` returns `nil` when the process have already exited.
Therefore, `Keyword.get(:messages)` may raise an error.

Reply:
fixed 9f63c88aedce6e7b810381f2e12f9f2978474395